src/Cake.Common/Tools/DotNetCore/DotNetCoreSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DotNetCore
+{
+    /// <summary>
+    /// Contains common settings used by <see cref="DotNetCoreTool{TSettings}" />.
+    /// </summary>
+    public class DotNetCoreSettings : ToolSettings
```

Comment:
I think that would be a good idea since this might change in the future. If it would then we break as little as possible for consumers of this alias.